BuildTools/swiftformat.sh

```diff
@@ -2,6 +2,15 @@
 
 if [ -z "$CI" ]; then
     export PATH="$PATH:/opt/homebrew/bin:/usr/local/bin"
+
     BASEDIR=$(dirname "$0")
-    xcrun --sdk macosx mint run -m "$BASEDIR/Mintfile" SwiftFormat --config "$BASEDIR/.swiftformat" .
+    MINT_FILE_PATH=$BASEDIR/Mintfile
+    SWIFT_FORMAT=SwiftFormat
```

Comment:
What benefit do we get from defining this variable? compared to just using SwiftFormat

same question for `SWIFT_LINT=SwiftLint`

Reply:
It's just to be easier to reference and update. For`MINT_FILE_PATH`, it's used in `l:9`, `l:12` and `l:15`. For `SWIFT_FORMAT`, it's used in `l:9` and `l:15`.

Reply:
Using `SWIFT_LINT` in 3 places or `SwiftLint` in 3 places doesn't really solve anything, until that app suddenly changes its own name, which seems unlikely :) 

I think it would be clearer to not have it as a variable, but not a big issue for me.

Reply:
not a deal-breaker for me either but I think it's extra